query/src/storages/fuse/meta/common.rs

```diff
@@ -40,3 +42,15 @@ where Self: Sized
 {
     const VERSION: u64 = V;
 }
+
+#[derive(Serialize, Deserialize, PartialEq, Clone, Debug)]
```

Comment:
```suggestion
#[derive(Serialize, Deserialize, PartialEq, Copy, Clone, Debug)]
```

So that we can avoid `compression.clone()`

Reply:
that is better!